Competition2017/src/competition/subsystems/feeder/SideFeederSubsystem.java

```diff
@@ -0,0 +1,170 @@
+package competition.subsystems.feeder;
+
+import org.apache.log4j.Logger;
+import com.ctre.CANTalon.FeedbackDevice;
+import com.ctre.CANTalon.TalonControlMode;
+import edu.wpi.first.wpilibj.Timer;
+import xbot.common.command.BaseSubsystem;
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.properties.BooleanProperty;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class SideFeederSubsystem extends BaseSubsystem {
+
+    private static Logger log = Logger.getLogger(FeederSubsystem.class);
+
+    private final XCANTalon masterMotor;
```

Comment:
let's rename this to `feederMotor', since there is no master/slave pair in this class